### PR TITLE
[EVNT-102] Strip protocol for http(s) components

### PR DESCRIPTION
**Pull request by CodeHeeler**

### CODE REVIEW

File: splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java [hunk 1]
```diff
@@ -136,14 +136,14 @@ private void configureHandler() throws Exception {
             // (see https://camel.apache.org/components/latest/http-component.html#_which_http_method_will_be_used).
             .choice()
                 .when(simple("${headers.metadata[url]} startsWith 'http://'"))
-                    .toD(http("$simple{headers.metadata[url]}/services/collector/event")
+                    .toD(http("$simple{headers.metadata[url].replaceFirst('^http:\/\/', '')}/services/collector/event")
```
[the remark this answers is not in the document]
Comment (CodeHeeler) [reply splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java:139]: it's necessary for regex, even if it is not for the java:
image

[the remark this answers is not in the document]
Comment (vkrizan) [reply splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java:139]: The slash notation is very usual in other languages like JavaScript, Ruby, or even sed. In Java this is provided within a string, therefore the slash is not used as a notation for regex. The page on screenshot has a switch to Java that will guide you as well.

Anyhow, if the slash needs to be escaped for whatever reason, it has to be double escaped like this `"\\/"`.

[the remark this answers is not in the document]
Comment (vkrizan) [reply splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java:139]: Please test out the code changes. It should give you a better insight.

**Review by vkrizan — COMMENTED** [review]
@CodeHeeler does the change work? Have you managed to test it out?

Code-wise looks good to me. Thx.

**Comment by CodeHeeler** [comment]
> @CodeHeeler does the change work? Have you managed to test it out?
> 
> Code-wise looks good to me. Thx.

will test before merging for sure; working on getting the whole testing setup going again and will make doc changes in the process.

**Pull request merged**